crates/matrix-sdk-ui/src/spaces/mod.rs

```diff
@@ -204,6 +212,98 @@ impl SpaceService {
         SpaceRoomList::new(self.client.clone(), space_id).await
     }
 
+    /// Returns all known direct-parents of a given space room ID.
+    pub async fn joined_parents_of_child(&self, child_id: &RoomId) -> Vec<SpaceRoom> {
```

Comment:
I'm not sure whether `child` is the right word here, instead of maybe `room`. I wonder if `child` will get conflated with `m.space.child` which is not really the case as the graph holds reverse relations as well 🤔